validation/views.py

```diff
@@ -444,6 +463,22 @@ def record_translation_judgement(request, phrase_id):
     return JsonResponse({"status": "ok"})
 
 
+@require_http_methods(["POST"])
+def record_audio_quality_judgement(request, recording_id):
+    # TODO: check that user is logged in
+    rec = get_object_or_404(Recording, id=recording_id)
+    print(recording_id)
+    judgement = json.loads(request.body)
```

Comment:
Looks like a stray `print()`?

If you like the following suggestion, you’ll need to [`from django.contrib.auth.decorators import login_required`](https://docs.djangoproject.com/en/3.1/topics/auth/default/#the-login-required-decorator)

```suggestion
@login_required()
@require_http_methods(["POST"])
def record_audio_quality_judgement(request, recording_id):
    rec = get_object_or_404(Recording, id=recording_id)
    judgement = json.loads(request.body)
```

Reply:
I removed the extra print--good catch!

I can't seem to get `login_required()` to work. It doesn't seem to realize that I am logged in and it ends up removing all permissions. Maybe we could pair on this a little?